Capri.Web/Controllers/ProposalController.cs

```diff
@@ -127,8 +132,15 @@ public async Task<IActionResult> Create([FromBody] ProposalRegistration registra
 
         [Authorize(Roles = "promoter")]
         [HttpPut("{id}")]
-        public async Task<IActionResult> Update(Guid id, [FromBody] ProposalRegistration newData)
+        public async Task<IActionResult> Update(
+            Guid id, 
+            [FromBody] ProposalRegistration newData)
         {
+            if(newData == null || !ModelState.IsValid)
+            {
+                return BadRequest("You provided invalid data");
```

Comment:
Same as here, impersonal form will be better.

Reply:
Done